iwyu_ast_util.cc

```diff
@@ -1079,6 +1080,15 @@ bool DeclsAreInSameClass(const Decl* decl1, const Decl* decl2) {
   return decl1->getDeclContext()->isRecord();
 }
 
+bool IsBuiltinFunction(
```

Comment:
This is correct formatting, but I think it would look nicer as:
```
bool IsBuiltinFunction(const clang::NamedDecl* decl,
                       const std::string& symbol_name) {
    ...
}
```

Does clang-format leave that alone?